Dockerfile.enhanced

```diff
@@ -4,7 +4,7 @@ WORKDIR /home/docsworker-xlarge
 COPY  config config/
 COPY package*.json ./
 COPY tsconfig*.json ./
-RUN npm ci
+RUN npm ci --legacy-peer-deps
```

Comment:
Is there a new dependency that warrants the `legacy-peer-deps` flag?

Reply:
I was running into build errors without this flag. I can test again to see if it works without needing it.

Reply:
Gotcha. If this is necessary, I'm fine with leaving in 👍